test/fixtures/test-runner/output/test-timeout-flag.js

```diff
@@ -1,19 +1,19 @@
-// Flags: --test-timeout=20
 'use strict';
```

Comment:
@jakecastelli I removed the flags here as they were not being used by the snapshots.
I moved them to the test-runner-output "flags".

Reply:
lgtm